sycl/unittests/pi/cuda/test_base_objects.cpp

```diff
@@ -118,6 +119,52 @@ TEST_F(DISABLED_CudaBaseObjectsTest, piContextCreatePrimary) {
       PI_SUCCESS);
 }
 
+TEST_F(DISABLED_CudaBaseObjectsTest, piContextCreatePrimaryFalse) {
+  pi_uint32 numPlatforms = 0;
+  pi_platform platform;
+  pi_device device;
+
+  ASSERT_EQ((Plugins[0].call_nocheck<detail::PiApiKind::piPlatformsGet>(
```

Comment:
What happens if there is more than one plugin? I guess it doesn't matter at the moment, because the unit tests are disabled, but we need to identify if the plugin is the CUDA one or not, this is OK for later.

Reply:
I guess we have a int id somewhere in the test for controlling which plugin to select

Reply:
This needs fixing but would be more work than this MR, so I adapted to what is already there.

Reply:
Makes sense, lets just make a note to fix later.